base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -503,11 +503,15 @@ pub fn create_lmdb_database(
     mmr_cache_config: MmrCacheConfig,
 ) -> Result<LMDBDatabase<HashDigest>, ChainStorageError>
 {
+    #[cfg(target_os = "windows")]
```

Comment:
It would be useful to put a comment here as to _why_ there's this bifurcation in the code.

Reply:
Removed bifurcation in the code